src/me/hkgumbs/tictactoe/main/java/Main.java

```diff
@@ -3,13 +3,15 @@
 import me.hkgumbs.tictactoe.main.java.configuration.Configuration;
 import me.hkgumbs.tictactoe.main.java.simulation.DefaultController;
 
+import java.util.Arrays;
 import java.util.NoSuchElementException;
 
 public class Main {
     private static final String USAGE = "usage: java -jar tictactoe-java.jar" +
             " [--size <number>]" +
             " [--padding <number>]" +
-            " [--humans|--minimax]";
+            " PLAYER1 PLAYER2" +
+            "\nPLAYER must be one of the following: human, minimax, naive";
```

Comment:
I usually try to have messages like this actually reference the [values](https://github.com/hkgumbs/tictactoe-java/pull/4/files#diff-3167962b7c4d1da8fdd12c5d45282207R15)  so the two don't get out of sync. At least you couldn't tell the user a string was valid when it isn't (which is very bad/frustrating for them), you could still forget to include it and have a valid choice not show up in the list (which is still bad, but less so imo).